sagemaker-fundamentals/create-training-job/create_training_job_encrypt_static_assets.ipynb

```diff
@@ -0,0 +1,1227 @@
+{
```

Comment:
<p>I hit the "Supply an existing CMK" error - I think you should add a text cell above this cell explaining your options below. I was confused and thought I needed to copy and paste something from the above cell. But actually it seems like I want to change <code>create_new_key = True</code>.  Maybe it should be set to True by default so customers can flow through the notebook unless they specifically choose to use an existing key?</p><p> </p><p>CMK still stands for customer master key - it should be replaced with "KMS key".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>added a markdown cell before it to explain what needs to happen </p>